frontend/src/pages/ImportCkan/SelectTagsScreen.jsx

```diff
@@ -0,0 +1,215 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { useSelector } from 'react-redux';
+
+import { LoadingButton } from '../../sharedComponents/LoadingButton';
+
+const WizardContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-content: space-around;
+  max-width: 50em;
+  min-width: 20em;
+  padding: 1em;
+  margin: 10px 0px;
+  background-color: white;
+  border-radius: 4px;
+  -webkit-box-shadow: 0 0.0625em 0.125em rgba(0,0,0,0.15);
+  -moz-box-shadow: 0 0.0625em 0.125em rgba(0,0,0,0.15);
+  box-shadow: 0 0.0625em 0.125em rgba(0,0,0,0.15);
+`;
+
+const TagListContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+`;
+
+const TagContainer = styled.div`
+    border-radius: 5px;
+    border: 1px solid #6666aa;
+    background-color: #ddddff;
+    padding: 0.5em;
+    margin: 1em;
+    height: 2em;
+    display: table-cell;
+    width: 20em;
+`;
+
+const ButtonPart = styled.div`
+    float: right;
+`;
+
+const LabelPart = styled.div`
+    position: absolute;
+    float: left;
+    line-height: 2em;
+`;
+
+const ImportButton = styled.div`
+    margin: 2px;
+    border-radius: 4px;
+    border: 1px solid #000000;
+    background-color: #aaaaaa;
+    padding: 0.2em;
+`;
+
+const NavigationContainer = styled.div`
+    display: flex;
+
```

Comment:
I'm guessing this is a leftover line from earlier in development
```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/725557/79563679-88244f80-80ad-11ea-8b1a-801e56cdc3a2.png)